language/go/generate_test.go

```diff
@@ -231,6 +231,30 @@ func TestGenerateRulesPrebuiltGoProtoRules(t *testing.T) {
 	}
 }
 
+func TestConsumedGenFiles(t *testing.T)  {
```

Comment:
Please add a quick comment explaining what is being tested here.

Also, it looks like some spaces are missing in the `GenerateArgs` literal below. Please run gofmt.